tests/framework/clients/filesystem.go

```diff
@@ -82,6 +83,7 @@ func (f *FilesystemOperation) Delete(name, namespace string) error {
 		return err
 	}
 
+	time.Sleep(time.Minute * 1)
```

Comment:
Instead of waiting for a whole minute, can we query every few seconds to see if the filesystem CR still exists? The integration tests are already so long I'd like to avoid long waits as much as possible.